### PR TITLE
Adding Matcher class for export and matching_score function for unit tests

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

File: evalme/matcher.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                            f"annotation={annotation}, prediction={prediction}, "
+                            f"Reason: {exc}",
+                            exc_info=True,
+                        )
+                if tasks > 0:
+                    scores[prediction['id']] = score / tasks
+        return scores
+
+
+def matching_score(annotations, predictions):
```
Comment (niklub) [inline evalme/matcher.py:95]: let's make atomic function `matching_score(annotation, prediction)` and some aggregator function `average_matching_score(annotations, predictions)` on top of it

**Review by niklub — CHANGES_REQUESTED** [review]

File: evalme/metrics.py [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+        return groups
+
+
+Metrics.register(
+    name='iou_bboxes',
+    form=None,
+    tag='RectangleLabels',
+    func=iou_bboxes,
+    desc='IOU for bounding boxes'
+)
```
Comment (niklub) [inline evalme/metrics.py:228]: Can we also add mAP as a bbox metric to use?

**Review by niklub — CHANGES_REQUESTED** [review]

File: evalme/metrics.py (hunk 2 above)
Comment (KonstantinKorotaev) [reply evalme/metrics.py:246]: Matcher.get_mAP_score()

File: evalme/matcher.py (hunk 1 above)
Comment (KonstantinKorotaev) [reply evalme/matcher.py]: Matcher.matching_score()

**Pull request merged**